python/src/sdf/pyParticleEmitter.cc

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "pyParticleEmitter.hh"
+
+#include <pybind11/pybind11.h>
+
+#include "sdf/ParserConfig.hh"
+
+#include "sdf/Box.hh"
+#include "sdf/Capsule.hh"
+#include "sdf/Cylinder.hh"
+#include "sdf/Ellipsoid.hh"
+#include "sdf/ParticleEmitter.hh"
+#include "sdf/Mesh.hh"
+#include "sdf/Plane.hh"
+#include "sdf/Sphere.hh"
+
+using namespace pybind11::literals;
+
+namespace sdf
+{
+// Inline bracket to help doxygen filtering.
+inline namespace SDF_VERSION_NAMESPACE {
+namespace python
+{
+/////////////////////////////////////////////////
+void defineParticleEmitter(pybind11::object module)
+{
+  // TODO(ahcorde) Add ParticleEmitterShape and SetParticleEmitterShape
```

Comment:
I can't find these functions in the C++ header. Do we need this TODO?

Reply:
https://github.com/gazebosim/sdformat/pull/999/commits/27a4ae4369eb9031c85655ee19b7c0d34b17e199